.env.example

```
@@ -1,2 +1,2 @@
 CF_EMAIL=your_cloudflare_account_email          # The email you use to login to Cloudflare
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)